Source/Core/Core/Analytics.h

```diff
@@ -40,6 +42,18 @@ class DolphinAnalytics
   // per-game base data.
   void ReportGameStart();
 
+  // Reports performance information. This method performs its own throttling / aggregation --
+  // calling it does not guarantee when a report will actually be sent.
+  //
+  // This method is NOT thread-safe.
```

Comment:
Nit: should this be next to the method instead of the struct (so it shows up in IntelliSense and perhaps other helper tooltip in $IDE)?